mini/Lexer.py

```diff
@@ -21,6 +21,7 @@ def advance(self):
             self.current_character = self.text[self.current_position.index]
         else:
             self.current_character = None
+        return self
```

Comment:
`return self` is not needed as we advance function is just used to make changes values of it class instance and advance function is only to be used inside class which always be refer by leading `self`. So, return self don't make any sense to me over here.